### DISCUSSION TITLE
docker network 'nextcloud-aio' disable internet connection

### DISCUSSION THREAD

**Discussion by sim-san** [nextcloud/all-in-one#699] [Questions] [answered]
I try to add a other service (LDAP) to the nextcloud-aio network. This works, but i lost the connection to the internet.
So i try to simplify the problem. I just run an network troubleshooting docker container and connect it to the nextcloud-aio network. Here II also lost the connection to the internet ?

```
ncadmin@vm-nextcloud:~$ sudo docker run --name netshoot --rm -it --network nextcloud-aio nicolaka/netshoot /bin/bash
bash-5.1# ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8) 56(84) bytes of data.
```

Do you have a hint ?
Is this somehow blocked from the Nextcloud AIO setup ?

**Comment by szaimen** [comment]
Do the nextcloud-aio containers have accesss to the internet?

**Reply by sim-san** [reply]
Yes. I tested nextcloud and database

```
ncadmin@vm-nextcloud:~$ sudo docker exec --user root -it nextcloud-aio-nextcloud bash
bash-5.1# ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8): 56 data bytes
64 bytes from 8.8.8.8: seq=0 ttl=114 time=1.731 ms
64 bytes from 8.8.8.8: seq=1 ttl=114 time=2.248 ms

ncadmin@vm-nextcloud:~$ sudo docker exec --user root -it nextcloud-aio-database bash
bash-5.1# ping 8.8.8.8
PING 8.8.8.8 (8.8.8.8): 56 data bytes
64 bytes from 8.8.8.8: seq=0 ttl=114 time=1.975 ms
64 bytes from 8.8.8.8: seq=1 ttl=114 time=1.667 ms
```

**Reply by szaimen** [reply]
then I don't think this is a problem of the docker network honestly...
Maybe you need to set a dns server for the ldap container manually?
e.g. with --dns=8.8.8.8 when starting the container...

**Reply by sim-san** [reply]
The interesting thing about this is that DNS works from the LDAP container. I guess because that possibly works via a cache in the nextcloud container.

**Reply by szaimen** [reply]
hm...

**Comment by szaimen** [comment, marked as answer]
You are right, this is a bug that wee need to fix. See https://github.com/nextcloud/all-in-one/issues/715

**Reply by szaimen** [reply]
See https://github.com/nextcloud/all-in-one/issues/715#issuecomment-1136268282 how to fix it

**Reply by sim-san** [reply]
Thanks. How do you find the solution ?

**Reply by szaimen** [reply]
I don't understand the question...
